blockchain/smart_contracts/sc_trusted.py

```diff
@@ -0,0 +1,229 @@
+"""
+Copyright 2016 Disney Connected and Advanced Technologies
+
+Licensed under the Apache License, Version 2.0 (the "Apache License")
+with the following modification; you may not use this file except in
+compliance with the Apache License and the following modification to it:
+Section 6. Trademarks. is deleted and replaced with:
+
+     6. Trademarks. This License does not grant permission to use the trade
+        names, trademarks, service marks, or product names of the Licensor
+        and its affiliates, except as required to comply with Section 4(c) of
+        the License and to reproduce the content of the NOTICE file.
+
+You may obtain a copy of the Apache License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the Apache License with the above modification is
+distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the Apache License for the specific
+language governing permissions and limitations under the Apache License.
+"""
+
+"""
+                **trusted smart contract module**
+smart contract code is run through this module under the assumption that the user is not delivering
+invalid or malicious code.
+"""
+
+__author__ = "Joe Roets, Brandon Kite, Dylan Yelton, Michael Bachtel"
+__copyright__ = "Copyright 2016, Disney Connected and Advanced Technologies"
+__license__ = "Apache"
+__version__ = "2.0"
+__maintainer__ = "Joe Roets"
+__email__ = "joe@dragonchain.org"
+
+import logging
+
+from blockchain.db.postgres import smart_contracts_db as sc_dao
+from blockchain.db.postgres import sub_to_db as sub_db
+
+import time
+import uuid
+
+
+def logger(name="verifier-service"):
+    return logging.getLogger(name)
+
+
+class SmartContractProvisioning(object):
+    def __init__(self, network=None, public_key=None):
+        # processing nodes network
+        self.network = network
+        # processing node's public key
+        self.public_key = public_key
+        self.rtsc = {"TT_SUB_REQ": self.subscription_request,
```

Comment:
Note:
To create:

- `smart_contracts.py`
- `reserved_sc.py`

Move all reserved SC into `reserved_sc.py`...
`smart_contracts.py` will contain generic provisioning and calling code.